app/assets/stylesheets/mixins/_mixins.scss

```
@@ -0,0 +1,3 @@
+@mixin primary-gradient {
```

How about renaming this file to `_background.scss`? so when we add more mixin later we can have them in separate files 💭 
And also should we name this more descriptive so that when we use it we know what it is for?

```suggestion
@mixin primary-gradient-bg {
```

or

```suggestion
@mixin primary-gradient-background {
```